### PR TITLE
Fix aclpolicy ownership

### PR DESCRIPTION
**Pull request by danifr**
I think this is pretty much self-explanatory

### CODE REVIEW

File: manifests/config/aclpolicyfile.pp [hunk 1]
```diff
@@ -64,12 +64,15 @@
...
-  $group          = 'rundeck',
-  $owner          = 'rundeck',
-  $properties_dir = '/etc/rundeck',
   $template_file  = "${module_name}/aclpolicy.erb",
 ) {
 
+  include ::rundeck
+
+  $owner          = $rundeck::group
+  $properties_dir = $::rundeck::config::properties_dir
```
Comment (jyaworski) [inline manifests/config/aclpolicyfile.pp]: This should be just rundeck::properties_dir, right?

Comment (danifr) [inline manifests/config/aclpolicyfile.pp]: Funny, we don't have it on init or params.
```
$ cat manifests/init.pp | grep properties_dir
# [*properties_dir*]
$ cat manifests/params.pp | grep properties_dir
$
```

Comment (jyaworski) [inline manifests/config/aclpolicyfile.pp]: ....really?

Well, looks like we need to do a further audit of parameters.

**Comment by jyaworski** [comment]
Tests fail.

**Comment by jyaworski** [comment]
Please rebase.

**Comment by jyaworski** [comment]
140 chars? Our limit is 328...

**Comment by igalic** [comment]
i'm pretty sure i've fixed this already…
@danifr another rebase should fix this…

but it doesn't explain the tweet size limit on our lines, as @jyaworski pointed out

**Comment by danifr** [comment]
@jyaworski is including this patch on https://github.com/voxpupuli/puppet-rundeck/pull/203

**Pull request closed without merging**